packages/xchain-util/src/async.test.ts

```diff
@@ -0,0 +1,65 @@
+import nock from 'nock'
+// eslint-disable-next-line ordered-imports/ordered-imports
+import { getAllInboundDetails, getInboundDetails, getMimirDetails } from './async'
+import { Chain } from './chain'
+
+describe('functions from Midgard', () => {
+  it('should return mimir details', async function () {
```

Comment:
Pls add another tests for `getMimirDetails` if both (not just one) Midgards are failing (404)